src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    const [property, value] = style.split(':', 2)
+      .map((item) => item.trim());
```

Comment:
The destructuring assignment is used correctly to extract the property and value. However, ensure that the `split` method's limit argument is necessary for the task requirements. If not, it could be omitted for simplicity.